### ISSUE TITLE
Lack of props which could prevent rendering Popconfirm buttons and renderContent prop

### ISSUE THREAD

**Issue by lukbrt** [ant-design/ant-design#27883]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Popconfirm component is not accessible out-of-the-box and I had to integrate it with focus-trap-react library to keep focus inside popover when it is open and allow closing the popover by pressing ESC. As a walkaround I had to wrap custom content and put it into title prop, as well as set default cancel/ok as hidden.

### What does the proposed API look like?
One of the following:
Provide prop which will override content of the popover (e.g. renderContent)
Add renderFooter prop (like in Modal component - footer prop overrides the default buttons)

**Comment by afc163** [comment]
I don't understand what's the Popconfirm looks like of in your application.

**Comment by lukbrt** [comment]
What I need to achieve is to wrap everything what is rendered inside:
<div class="ant-popover-inner" role="tooltip"></div>
With <FocusTrap> component which comes from focus-trap-react library. What the library does is to keep focus inside popconfirm when it is open and allow closing it by pressing ESC. I didn't found any direct ways to achieve this in antd popconfirm component, so I passed the following props (where d-none class is simply display: none;):
icon={<></>}
okButtonProps={{ className: 'd-none' }}
cancelButtonProps={{ className: 'd-none' }}
After that I rendered normal popover content (title, icon and yes/no buttons) by passing JSX to title prop. After that I achieved accessibly popconfirm, but I still have the default buttons which shoudn't render at all.
